tests/testAutopas/tests/selectors/tuningStrategy/TuningStrategyTest.cpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file TuningStrategyTest.cpp
+ * @author Jan Nguyen
+ * @date 27.04.20
+ */
+
+#include "TuningStrategyTest.h"
+
+#include <gmock/gmock-matchers.h>
+#include <gmock/gmock-more-matchers.h>
+
+TEST_P(TuningStrategyTest, testSearchSpaceEmpty) {
+  auto noInterval = autopas::NumberSetFinite<double>({});
+  EXPECT_THROW(autopas::TuningStrategyFactory::generateTuningStrategy(
+                   GetParam(), {}, noInterval, {}, {}, {}, 42, autopas::AcquisitionFunctionOption::expectedDecrease),
```

Comment:
for all tests:
Please save the value of `GetParam()` to a variable with a meaningful name before using it. This greatly improves readability because otherwise, one has to first scroll all the way down to see what the param actually is.